Cartesian_kernel/include/CGAL/Cartesian/Vector_2.h

```diff
@@ -50,7 +50,10 @@ class VectorC2
   VectorC2() {}
 
   VectorC2(const FT &x, const FT &y)
-    : base(CGAL::make_array(x, y)) {}
+    : base{x, y} {}
+
+  VectorC2(FT&& x, FT&& y)
+    : base{std::move(x), std::move(y)} {}
```

Comment:
IIUC we use 2 overloads instead of perfect forwarding + make_array because make_array doesn't work well enough?